### PR TITLE
Added grant request for Jur's research on token morphism

### PR DESCRIPTION
**Pull request by tyagishuchi1**
#### Project Abstract
Research paper regarding the SEC DOT regulatory process and outcomes and recommendations for Dotsama and the wider Web3 ecosystem

#### Grant [level](https://github.com/w3f/Grants-Program#level_slider-levels)

- [ ] **Level 1**:  Up to $10,000, 2 approvals
- [x] **Level 2**:  Up to $30,000, 3 approvals
- [ ] **Level 3**:  Unlimited, 5 approvals (for >$100k: Web3 Foundation Council approval)

#### Application Checklist

- [x] The [application template](https://github.com/w3f/Grants-Program/blob/master/applications/application-template.md) has been copied and aptly renamed (`project_name.md`).
- [x] I have read the [application guidelines](https://github.com/w3f/Grants-Program/blob/master/docs/Support%20Docs/grant_guidelines_per_category.md).
- [x] A BTC, Ethereum (USDC/DAI) or Polkadot/Kusama (aUSD/USDT) address for the payment of the milestones is provided in the application.
- [x] The software delivered for this grant will be released under an open-source license specified in the application.
- [x] The initial PR contains only one commit (squash and force-push if needed).
- [x] The grant will only be announced once the first milestone [has been accepted](https://github.com/w3f/Grant-Milestone-Delivery#process) (see the [announcement guidelines](https://github.com/w3f/Grants-Program/blob/master/docs/Support%20Docs/announcement-guidelines.md)).
- [ ] I prefer the discussion of this application to take place in a private Element/Matrix channel. My username is: `@_______:matrix.org` (change the homeserver if you use a different one)

### CODE REVIEW

**Review by Noc2 — CHANGES_REQUESTED** [review]
Thanks a lot for the application. I will share it internally to get some initial feedback. Apart from this, I noticed that the delivery of milestone 1 contains an “internal draft.” However, in order to be able to evaluate (see https://github.com/w3f/Grant-Milestone-Delivery) this via our grants program, it would be nice if this “internal draft” could already be publicly shared. It doesn’t need to be shared via public channels like Twitter etc., but it should be publicly accessible via a link, e.g. on GitHub.

**Comment by taqtiqa-mark** [comment]
Disclosure: IANAL. Nor do I work in the legal space. I am a contractor who contemplated entering the "blockchain analysis" market segment, but am contemplating a pivot for the reasons the applicants cite in submitting their 'low-ball' application.

@Noc2 this is an important piece of work that should be funded, but not without changes that are largely the W3F's to drive. 

TLDR;

1. Ask the team to resubmit with a budget that is commercially sustainable/realistic.
2. If that budget is beyond the W3F's capacity or remit, either:
    a. Offer to assist them in carving/trailblazing a pathway for funding this from Dotsama Treasuries. If that fails or proves impractical,
    b. the W3F apply to the Dotsama treasuries for a "top-up" so that such applications can be funded and this and other important information/knowledge is made generally available.  If the amount needed is judged by the dotsama community to be excessive,
    c. the W3F prepare a governance proposal that burning DOT cease or is slowed until the dotsama treasuries are sufficiently funded to support the W3F.

The problem is this:

> The team involved decides to work on a cost-reimbursement basis. If market rates were to be applied for professional or research activities, the cost would be prohibitively high.

This seems an appealing offer but on reflection (below) will set back the production of such research/information, for one or more of these reasons:

1. Such applications (intentionally or not) ultimately have the effect of driving similar groups/firms out of business.  This reduces competition **and** supply. This means such information eventually becomes harder to come by **and** more expensive.
2. Funding sustainable applications, has the effect of sustaining applications. As the applicants note this area is fluid, and similar updates will be required in the future. 
3. The applicants are working in large, developed markets for legal and information services such that they are able to cross-subsidize the W3F as they propose.  Not all dotsama developers live in, nor target, the US/EU markets.  Yet they too will need similar information for their context. However, many of the countries/regions don't have developed markets for legal and information services such that they can offer the W3F these subsidies.

I'll stop there. Hope you think twice about going down this path. In the long/medium/short run you aren't actually doing anyone a favor by accepting such subsidies.

**Comment by Ale1648** [comment]
Hi @taqtiqa-mark, @Noc2,

Regarding the internal draft as milestone 1, we will modify the proposal according to the suggestion received. 

@Mark, I was pleased to read your notations and agree. The issue you raise is important for the whole ecosystem. On several occasions, I have thought the same thing and have also written on the Polkadot forum with proposals to improve Treasury 2. 

Polkadot and governance in general should work in the opposite way to how public treasuries are often run: bids should be competitive with the private sector so as to attract top players, with the right incentive system. Qualified teams do not want to work on unaffordable economic terms and, in addition, be subject to public scrutiny that generates time-consuming reporting activities. 

A different approach produces the effects you mention and in addition:

- Failure to spend treasury efficiently due to lack of viable proposals from qualified teams.
- Progressive degrowth of the ecosystem.
- Only teams already involved in Dotsama can afford to contribute on economic terms that in themselves are not sustainable or affordable according to standard market rates.

In the specific case, Mark, as Jur we did some reflection and decided to:

- Support the ecosystem with immediate effect by proposing a cost-reimbursement activity. 

- In parallel, to contribute to a redefinition of Treasury v2 in a direction similar to the one you mentioned, so as to make an overall critical improvement in the Polkadot context.

In this particular case, and given the timing, we thought we would sacrifice something in the immediate term to make sure that we could deliberate quickly on something that is needed for the industry. Check out this series of proposals for Treasury 2: https://forum.polkadot.network/t/a-better-treasury-system/291/17?u=ale.


For this particular case, I propose a **third solution, perhaps a middle ground**:

1. We will do the research under the proposed economic conditions. Let's see in concrete terms how long it takes and see what result we can deliver.  
2. Once we deliver, it is up to the community to define a Tip to reward the work done. 
3. We will use this direct experience to reformulate the proposals for Treasury 2. In my opinion, it is difficult to revise the terms of this proposal in the absence of a more advanced grant management structure for the Polkadot and/or Web3 Foundation treasury that includes: dedicated and qualified _project managers_  (liaisons, in our proposal) who assist the team competently with a set of powers and prerogatives defined ex ante, a system of _escrow_ of the total amount allocated, _dispute resolution systems_ in case of need etc..

How does this sound? 

Thanks in advance, best,

Alessandro

**Comment by tyagishuchi1** [comment]
> Thanks a lot for the application. I will share it internally to get some initial feedback. Apart from this, I noticed that the delivery of milestone 1 contains an “internal draft.” However, in order to be able to evaluate (see https://github.com/w3f/Grant-Milestone-Delivery) this via our grants program, it would be nice if this “internal draft” could already be publicly shared. It doesn’t need to be shared via public channels like Twitter etc., but it should be publicly accessible via a link, e.g. on GitHub.

Thankyou @Noc2. The new commit reflects the requested changes.

**Review by Noc2 — CHANGES_REQUESTED** [review]
Thanks again for the application and all the effort you put into this. We discussed this internally and concluded that it would be great if you could apply for treasury funding instead. On the one hand, this grants program is currently focused on open-source software development and very few technical research applications. We have never evaluated any legal applications, nor does the grants committee have any expertise in this field. On the other hand, it would be nice if the on-chain treasury funds this kind of research, and it provides you the opportunity to ask for more than to cover your costs.
Additionally, the treasury pays upfront. So I would close the application for now, and if you reach out to me via email: david@web3.foundation, I can put you in touch with Otar and Raul, who can help you with the treasury application. If the treasury application is unsuccessful, we always have the option to reopen this application. Feel free to let me know if you have any questions.

**Comment by tyagishuchi1** [comment]
Thank you for the clarification and guidance @Noc2 , team really appreciates it. We will reach out soon with a proposal for the treasury. 🙌

**Comment by Polkadot-Forum** [comment]
This pull request has been mentioned on **Polkadot Forum**. There might be relevant details there:

https://forum.polkadot.network/t/polkadot-purple-paper-token-morphism-guidelines/2023/1
